app/assets/javascripts/darkswarm/cookies_policy/cookies_policy.html.haml

```diff
@@ -0,0 +1,89 @@
+%h2
```

Comment:
When you introduce new user interfaces, it's helpful to add some screenshots to your pull request.

Reply:
Yes, I have now added links to the screenshots as a comment to this PR.

Reply:
yes, that makes sense. I have added links to all the screenshots below as a comment to this PR.

Reply:
thanks!